devices/gd32e230.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 The gd32-rs authors.
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+_svd: ../svd/gd32e230.svd
+
+_modify:
+  name: "GD32E230"
+
+RCU:
+  VKEY:
+    _modify:
+      KEY:
+        access: write-only
```

Comment:
You have this in `common_patches/gd32e23x.yaml` already, you don't need to repeat it here.

Reply:
oh thanks, forgot to delete